backend/src/controllers/vehicle-controler.ts

```
@@ -0,0 +1,42 @@
+    insurance: boolean,
+    customerID: number,
+    vehicleID: number,
+  ): Promise<Vehicle[] | null> {
+    const query: string =
+      "INSERT INTO Reservations (StartDate, EndDate, Insurance, CustomerID, Vehicle)" +
+      "VALUES (?, ?, ?, ?, ?)";
+    const values = [startDate, endDate, insurance, customerID, vehicleID];
+    const [result]: any = await db.query(query, values);
+    return result.length ? result[0] : null;
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)